.jshintrc

```diff
@@ -37,6 +37,10 @@
         "before": true,
         "it": true,
         "expect": true,
-        "angular": false
+        "angular": false,
+        "inject": true,
+        "spyOn": true,
+        "successfulPromise": true,
+        "fdescribe": true
```

Comment:
remove `fdescribe` from jshint - for us is like with `console.log` - you can use `consol log` when you're developing but when you commit changes you have to remove it from code;